packages/sandcastle/gallery/box/main.js

```diff
@@ -0,0 +1,38 @@
+import * as Cesium from "cesium";
+
+/* eslint-disable no-unused-vars */
+
+const viewer = new Cesium.Viewer("cesiumContainer");
+
+const blueBox = viewer.entities.add({
```

Comment:
There were a number of sandcastles that follow this structure of defining multiple examples and assigning them to variables that never actually get used. I thought about disabling `no-unused-vars` for _all_ sandcastles but I do think it's a valid error we want to avoid. Instead I disabled it for these files specifically. I do think it's handy to have examples like these so it's easy for people to play with them if they want.